test/integration/post_index_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
- [ ] [OPTIONAL] Kindly move the integration folder to the spec folder instead of the test folder and change the name of each file to end with `spec` instead of `test` in order for the tests to run without any problem. The screenshot provided for the running test shows only the old tests exists in the spec folder, not the new ones.